test/units/cli/test_galaxy.py

```diff
@@ -444,3 +457,261 @@ def test_template_ignore_similar_folder(self):
 
     def test_skeleton_option(self):
         self.assertEquals(self.role_skeleton_path, context.CLIARGS['role_skeleton'], msg='Skeleton path was not parsed properly from the command line')
+
```

Comment:
I think that at this point I'd like all the new additions below this line to be moved to a separate module for better maintainability.

Reply:
I disagree, I've been moving some common functionality to another file and keeping `galaxy.py` just the stub that invokes the new actions. Because of this I don't think it is really necessary to split test_galaxy into more than 1 file.